app/views/layouts/base.html.erb

```diff
@@ -74,7 +74,7 @@
         local_assigns[:user_main_tag] == false ? 'div' : 'main',
         class: 'site-wrap bg-primary-lighter',
         id: local_assigns[:user_main_tag] == false ? nil : 'main-content',
-      ) do %>  
+      ) do %>
```

Comment:
❤️ thanks for catching

Reply:
Thanks to VSCode's `"files.trimTrailingWhitespace": true` 😄